base/install_node

```diff
@@ -79,17 +79,22 @@ function verifyBinaryExists(version, cb) {
 
 
 function downloadAndInstallVersion(version, cb) {
-  var downloadUrl = 'https://storage.googleapis.com' +
-    '/gae_node_packages/node-' + version + '-linux-x64.tar.gz';
   var destPath = '/nodejs';
-  var cmd = 'curl ' + downloadUrl + '| tar xzf - -C ' + destPath +
-    ' --strip-components=1';
+  var baseUrl = 'https://storage.googleapis.com/gae_node_packages';
+  var cmd = '/opt/nodejs/verify_install_node ' + baseUrl +
+    ' ' + version + ' ' + version + '-SHASUMS256.txt.asc ' + destPath +
+    ' WARN_ON_ERROR';
```

Comment:
The SHASUMS files are not being mirrored yet.  The reason why the command doesn't just specify the full SHASUMS URL is because of differences in the directory structure of the bucket and node.js.org/dist.